### PR TITLE
[Ready for Publish] Chinese - Accessibility Principles

### PR DESCRIPTION
**Pull request by ruoxiran**

### CODE REVIEW

**Comment by shawna-slh** [comment]
@ruoxiran , Per [Translator Link permissions](https://www.w3.org/WAI/about/translating/#links), the link cannot go to a home page. Links should only go to information about the translator.

I edited it so it is corrected for review. Please confirm with the translator that this is OK:

https://lisongfeng.cn -> https://lisongfeng.cn/index.php/start-page.html

File: index.zh-hans.md [hunk 1]
```diff
@@ -0,0 +1,724 @@
...
+
+标准在为每个组件定义无障碍需求方面扮演着重要的角色。一些无障碍需求很容易满足，但是了解残疾人如何使用Web的基本知识有助于更有效地实现这些需求。无障碍的某些方面需要更多的技术技能，也需要关于人们如何使用Web的高级知识。在所有情况下，[在整个web项目中早期让用户参与](/test-evaluate/ involved -users/)将使您的工作更好、更容易。
+
+W3C的Web无障碍推进(WAI)提供了一组被国际公认为Web无障碍标准指南。这些包括:
+
+-   **[Web内容无障碍指南 (WCAG)](/standards-guidelines/wcag/)**
+-   **[用户代理无障碍指南 (UAAG)](/standards-guidelines/uaag/)**
+-   **[无障碍创作工具指南 (ATAG)](/standards-guidelines/atag/)**
+
+还有一个用于**[无障碍富网络应用(WAI-ARIA)]的WAI规范(https://www.w3.org/WAI/intro/aria.php)** ，其中包括使用Ajax、JavaScript和相关web技术开发的动态内容和高级用户界面控件。
```
Comment (shawna-slh) [inline index.zh-hans.md]: Needs link fix so there are no characters or space between the closing bracket](and the opening parentheses:
``` **[ ... (WAI-ARIA)](https://www.w3.org/WAI/intro/aria.php)** ```
Also "WAI specification" not bold.

so maybe: 

```**[无障碍富网络应用(WAI-ARIA)](https://www.w3.org/WAI/intro/aria.php)** 的WAI规范 ```

?

**Review by shawna-slh — COMMENTED** [review]
Looks good with one change noted. Of course, I cannot read it. ;-) I did QA what I could.

**Comment by ruoxiran** [comment]
> @ruoxiran , Per [Translator Link permissions](https://www.w3.org/WAI/about/translating/#links), the link cannot go to a home page. Links should only go to information about the translator.
> 
> I edited it so it is corrected for review. Please confirm with the translator that this is OK:
> 
> https://lisongfeng.cn -> https://lisongfeng.cn/index.php/start-page.html

Yes, the translator OK with that. thank you.

File: index.zh-hans.md (hunk 1 above)
Comment (ruoxiran) [reply index.zh-hans.md]: I fixed this issue, thank you for review.

**Comment by ruoxiran** [comment]
This doc has been reviewed, ready to publish, thanks.

**Pull request merged**